scratch2/blocks.js

```diff
@@ -596,17 +593,17 @@ class ScriptView {
   }
 
   measure() {
-    for (var i = 0; i < this.blocks.length; i++) {
-      this.blocks[i].measure()
+    for (const block of this.blocks) {
+      block.measure()
     }
   }
 
   draw(inside) {
     var children = []
     var y = 0
     this.width = 0
-    for (var i = 0; i < this.blocks.length; i++) {
-      var block = this.blocks[i]
+    let block
```

Comment:
Hmm, what is this? if this is to grab the last block, we should just `this.blocks[this.blocks.length - 1]` imo.